src/components/pages/SingleEventPage.jsx

```diff
@@ -2,35 +2,90 @@ import { useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { useParams } from 'react-router-dom';
 import { fetchEventById } from '../../redux/events/operations_';
-import { getEvents } from '../../redux/selectors';
+import { getCurrentEvent, getLang } from '../../redux/selectors';
+import { Icon } from '@iconify/react';
+
 
 export const SingleEventPage = () => {
   const { id } = useParams();
   const dispatch = useDispatch();
+  const currentEvent = useSelector(getCurrentEvent);
+  const lang = useSelector(getLang);
 
   useEffect(() => {
     dispatch(fetchEventById(id));
   }, [dispatch, id]);
 
-  const { currentEvent_ } = useSelector(getEvents);
-  console.log(currentEvent_);
-
-  return currentEvent_ ? (
-    <>
-      <h1>
-        I am SingleEventPage of{' '}
-        <span className="text-red-600 font-bold">{currentEvent_.title.en}</span>
-      </h1>
-      <p className="text-red-600 font-bold">
-        Vadym, update me using 'currentEvent' redux state and this{' '}
-        <a
-          target="blank_"
-          href="https://www.eventbrite.ca/e/salsa-bachata-international-artist-weekender-sep-14-15-2024-tickets-892119351997?aff=erelexpmlt"
-          className="underline underline-offset-2"
-        >
-          template
-        </a>
-      </p>
-    </>
-  ) : null;
+  if (!currentEvent) {
+    return <div>Loading...</div>;
```

Comment:
I live it as is for now. But later replace "Loading..." by some EmptyPage. Like: "There is no data, try again later or contact to admin"